internal/config/config.go

```diff
@@ -25,36 +27,10 @@ import (
 
 var (
 	// Logrus structured logging setup
-	logFields = logrus.Fields{
+	cfgLog = logrus.WithFields(logrus.Fields{
 		"app":       "openmatch",
 		"component": "config",
-	}
-	cfgLog = logrus.WithFields(logFields)
-
-	// Map of the config file keys to environment variable names populated by
```

Comment:
Do we have a replacement connection config ready for Redis yet? If not I wouldn't move this. The reason we kept around the old code was there's a lot of scaffolding we didn't want to have to rewrite. This is part of that scaffolding.